diag_manager/diag_yaml_format.md

```diff
@@ -524,4 +525,63 @@ diag_files:
   - *name
   - variable_name: var773
 ```
+### 7. Improving Performance at High Resolutions
+For very high resolutions (e.g., C3072L65), performance can be greatly improved by using NETCDF-4 with collective MPI writes. Tests have shown up to a 70% speedup when writing hourly diagnostic output. This approach also simplifies the workflow, as it does not use the IO domain and does not create distributed files. To enable this feature, set `use_collective_writes: True` for each file. For example:
+```yaml
+title: test_none
+base_date: 2 1 1 0 0 0
+diag_files:
+- file_name: test_none
+  freq: 6 hours
+  time_units: hours
+  unlimdim: time
+  module: ocn_mod
+  reduction: none
+  kind: r4
+  use_collective_writes: True
+  chunksizes: 96, 8, 1, 1, 1
+  varlist:
+  - var_name: var4
+```
+Chunking can also have a major impact on performance. You can set chunk sizes for all variables in a file using the `chunksizes` key at the file level. By default, the diag manager will choose chunk sizes equal to the compute domain for domain-decomposed dimensions (e.g., for C3072L65 and a layout of 32,64, the chunksizes would be 96,48). For non-domain-decomposed dimensions, the chunk size is set to the dimension length. For unlimited dimensions, the chunk size is always 1. For example:
```

Comment:
Just to be clear, there is no chunking for the K-dim?  If not, that should be specified in the documentation and the  example chunksize would be 96,48,64 (65) depending on layer (level) and assuming we are explicitly setting the k-dim chunksize.